sui-execution/latest/sui-adapter/src/static_programmable_transactions/mod.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    execution_mode::ExecutionMode, execution_value::ExecutionState, gas_charger::GasCharger,
+};
+use move_trace_format::format::MoveTraceBuilder;
+use move_vm_runtime::move_vm::MoveVM;
+use std::{cell::RefCell, rc::Rc, sync::Arc};
+use sui_protocol_config::ProtocolConfig;
+use sui_types::{
+    base_types::TxContext, error::ExecutionError, execution::ResultWithTimings,
+    metrics::LimitsMetrics, transaction::ProgrammableTransaction,
+};
+
+// TODO we might replace this with a new one
+pub use crate::programmable_transactions::linkage_view::LinkageView;
+
+pub mod env;
+pub mod execution;
+pub mod loading;
+pub mod spanned;
+pub mod typing;
+
+macro_rules! better_todo {
```

Comment:
I'm confused about this/how it's better than just `todo!`... but more curious than anything else as eventually it'll all be gone anyways.